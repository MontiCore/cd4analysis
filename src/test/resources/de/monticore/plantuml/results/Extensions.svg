<?xml version="1.0" encoding="us-ascii" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentStyleType="text/css" height="303px" preserveAspectRatio="none" style="width:208px;height:303px;background:#FFFFFF;" version="1.1" viewBox="0 0 208 303" width="208px" zoomAndPan="magnify"><defs/><g><rect fill="none" height="27.6094" id="_legend" style="stroke:#000000;stroke-width:1.0;" width="106" x="47" y="12"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="96" x="52" y="31.5332">.Extensions CD</text><!--class A--><g id="elem_A"><rect codeLine="19" fill="#FFFFFF" height="42.7031" id="A" style="stroke:#181818;stroke-width:0.5;" width="60" x="89" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="12" font-style="italic" lengthAdjust="spacing" textLength="58" x="90" y="77.0664">&#171;abstract&#187;</text><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="9" x="114.5" y="94.2363">A</text></g><!--class B--><g id="elem_B"><rect codeLine="21" fill="#FFFFFF" height="27.6094" id="B" style="stroke:#181818;stroke-width:0.5;" width="15" x="86.5" y="162.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="9" x="89.5" y="182.1426">B</text></g><!--class C--><g id="elem_C"><rect codeLine="23" fill="#FFFFFF" height="27.6094" id="C" style="stroke:#181818;stroke-width:0.5;" width="16" x="137" y="162.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="10" x="140" y="182.1426">C</text></g><!--class D--><g id="elem_D"><rect codeLine="25" fill="#FFFFFF" height="27.6094" id="D" style="stroke:#181818;stroke-width:0.5;" width="16" x="61" y="250.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="10" x="64" y="270.1426">D</text></g><!--class E--><g id="elem_E"><rect codeLine="27" fill="#FFFFFF" height="27.6094" id="E" style="stroke:#181818;stroke-width:0.5;" width="15" x="112.5" y="250.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="9" x="115.5" y="270.1426">E</text></g><!--reverse link A to B--><g id="link_A_B"><path d="M108.17,122.6094 C104.35,136.9094 100.31,152.0094 97.53,162.4194 " fill="none" id="A-backto-B" style="stroke:#181818;stroke-width:1.0;"/><polygon fill="none" points="101.52,120.3694,113.45,102.8594,115.05,123.9894,101.52,120.3694" style="stroke:#181818;stroke-width:1.0;"/></g><!--reverse link A to C--><g id="link_A_C"><path d="M130.26,122.6094 C134.24,136.9094 138.44,152.0094 141.33,162.4194 " fill="none" id="A-backto-C" style="stroke:#181818;stroke-width:1.0;"/><polygon fill="none" points="123.38,123.9994,124.77,102.8594,136.87,120.2494,123.38,123.9994" style="stroke:#181818;stroke-width:1.0;"/></g><!--reverse link B to D--><g id="link_B_D"><path d="M84.61,209.9094 C80.52,223.9694 75.96,239.6494 72.83,250.4294 " fill="none" id="B-backto-D" style="stroke:#181818;stroke-width:1.0;"/><polygon fill="none" points="77.9,207.9194,90.2,190.6694,91.34,211.8294,77.9,207.9194" style="stroke:#181818;stroke-width:1.0;"/></g><!--reverse link B to E--><g id="link_B_E"><path d="M103.76,209.9094 C108.01,223.9694 112.76,239.6494 116.01,250.4294 " fill="none" id="B-backto-E" style="stroke:#181818;stroke-width:1.0;"/><polygon fill="none" points="97.04,211.8394,97.95,190.6694,110.44,207.7894,97.04,211.8394" style="stroke:#181818;stroke-width:1.0;"/></g><text fill="#888888" font-family="sans-serif" font-size="10" lengthAdjust="spacing" textLength="200" x="0" y="293.5996">generated with MontiCore using PlantUML</text><!--SRC=[RP31JiCm38RFpLFu2O07g2gjxGv8J9f0Y7EQcjPQcbIEAnXexqxIT4WJv91Od_tRlrC6ELkiL66i3g4bFR2Uu5iL6QnHKMjp6jWlhc-yzGnl8mccn7Dcx93XGHK0btHP7D3rKGhh-L-TSxSD4hceydxpD5AFO8YDnHpZDCiP9fmwvA1YUJYHctLolBhTYJ1rY-2ZyUw5lZ3stCo8du5f62MgxlQVWYwGTm6QLgLqhb9vvnsK9UWk26iZK5LgupLW4lQnuegQlwZzHVKLxMyGEedRldkVdZWMMGlsy44omi4xeVWV24iWDy3HQYUlXwSV]--></g></svg>
