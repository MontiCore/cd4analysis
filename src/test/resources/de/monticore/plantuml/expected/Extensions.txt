@startuml
  skinparam classAttributeIconSize 0
  skinparam legend {
    BorderColor black
    BackGroundColor white
  }
  legend top right
    .Extensions CD
  end legend
  class A {
    {field} int a
  }

  class B extends A {
    {field} int b
  }

  class C extends A {
    {field} int c
  }

  class D extends B {
    {field} int d
  }

  class E extends B {
    {field} int e
  }

  center footer generated with MontiCore using PlantUML
@enduml
